.idea/misc.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
Can we figure out why the project is downgrading the language level? I think we should be using JDK_1_8. This will cause us problems if two pull requests differ in JDK for this file.